scripts/Editor/DllManipulatorWindowEditor.cs

```diff
@@ -6,13 +6,22 @@ namespace UnityNativeTool.Internal
 {
     public class DllManipulatorWindowEditor : EditorWindow
     {
+        private static EditorWindow window;
+
         [MenuItem("Window/Dll manipulator")]
         static void Init()
         {
-            var window = GetWindow<DllManipulatorWindowEditor>();
+            window = GetWindow<DllManipulatorWindowEditor>();
             window.Show();
         }
 
+        [NativeDllLoadedTrigger]
+        [NativeDllAfterUnloadTrigger]
+        public static void RepaintAll()
+        {
+            if(window) window.Repaint();
```

Comment:
Out of curiosity, repainting an editor inside a window doesn't also make the window repaint?

And as a nit, I don't use one line control blocks in this project (and anywhere else). So just an enter here.

Reply:
Yes I tested it today with the new changes 74c1c96f19b432179a9780be3f3e0ac42a71aff8 and you seem to have to also repaint the window.

>So just an enter here

It's no longer in there but sure.